src/kvstore/raftex/RaftexService.cpp

```diff
@@ -58,12 +59,15 @@ void RaftexService::waitUntilReady() {
 }
 
 
-void RaftexService::initThriftServer(std::shared_ptr<folly::IOThreadPoolExecutor> pool,
-                                     uint16_t port) {
+void RaftexService::initThriftServer(std::shared_ptr<folly::IOThreadPoolExecutor> ioPool,
+    std::shared_ptr<folly::IOThreadPoolExecutor> acceptPool,
+    uint16_t port) {
     LOG(INFO) << "Init thrift server for raft service.";
     server_->setPort(port);
-    if (pool != nullptr) {
-        server_->setIOThreadPool(pool);
+    if (ioPool != nullptr && acceptPool != nullptr) {
+        server_->setIOThreadPool(ioPool);
+        server_->setIOThreadPool(acceptPool);
```

Comment:
`setIOThreadPool` twice ?  Maybe it's `setAcceptExecutor`

Reply:
good point, thx.